.bazelrc

```diff
@@ -1,4 +1,3 @@
 # Force the tests to not run in sandboxed mode, since it can introduce
 # errors and flakes.
 test --spawn_strategy=local
-build --macos_minimum_os=10.9
```

Comment:
not to add more thrash, but I guess you would probably still prefer this here, unless setting them on the build test targets is enough, just in case people run the tests locally?